bin/build.sh

```diff
@@ -1,43 +1,59 @@
 #!/bin/bash
 
+# Makes it so that all commands are displayed as they get executed
 set -x
 
 wca_url="https://worldcubeassociation.org/"
+stylesheet=$(realpath "./assets/style.css")
```

Comment:
This assumes that we're running the script from the root folder of that repository; this seems fine because we're actually assuming something similar below.
However if at some point you want to refer a file relative to the script's location, you can consider using `$(dirname "$0")` to get the script's folder and build the path from that.